packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -40,13 +44,41 @@ export const TransactionForm = ({ propertyAddress }: Props) => {
   const { data: propertyAuthData } = useGetPropertyAuthenticationQuery({ variables: { propertyAddress } })
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const metricsAddress = useMemo(() => propertyAuthData?.property_authentication[0]?.metrics, [propertyAuthData])
+
   const checkWithdrawable = useCallback(
     () =>
-      Promise.all([getWithdrawalStatus(propertyAddress), getBlockNumber()]).then(([withdrawStatus, blockNumber]) =>
-        Boolean(withdrawStatus && blockNumber && withdrawStatus <= blockNumber)
-      ),
-    [propertyAddress]
+      Promise.all([getWithdrawalStatus(propertyAddress), getBlockNumber()]).then(([withdrawStatus, blockNumber]) => {
+        setWithdrawable(Boolean(withdrawStatus && blockNumber && withdrawStatus <= blockNumber))
```

Comment:
I quit returning the value in this function and changed it to manage the value in state.